packages/design-system-react/src/components/button-link/ButtonLink.types.ts

```diff
@@ -0,0 +1,47 @@
+import type { ButtonBaseProps } from '../button-base';
+import { ButtonBaseSize } from '../button-base';
+
+export enum ButtonLinkSize {
+  Sm = ButtonBaseSize.Sm,
+  Md = ButtonBaseSize.Md,
+  Lg = ButtonBaseSize.Lg,
+  /**
+   * Inherits size from parent, removes height/padding, displays inline
+   */
+  Inherit = 'inherit',
```

Comment:
This is totally my bad, but can we align on the naming of this size? Right now in mobile it's "Auto", and, imo, auto makes more sense than "Inherit", since even the classname youre using is "auto"
Also Inherit doesnt exist on mobile

Reply:
Good catch! I agree this aligns well with mobile and feels more intuitive. I've updated it here: https://github.com/MetaMask/metamask-design-system/pull/298/commits/2a84e36e13bddd0230c2bb3e01c38990fbc5f57f

I've also confirmed that there are no other instances of `ButtonLinkSize.Inherit` in the codebase.  

![Uploading Screenshot 2025-01-06 at 3.23.05 PM.png…]()